data/pages-content/mentorship.md

```diff
@@ -15,7 +15,7 @@ At the moment we are looking for interested mentors who would like to help our m
 
 ## 1. One-to-One Mentorship Programme
 
-### Open between [Dates - TBA]
+### Open between 15 February - 26 February 2024
```

Comment:
I think we need to wait until the CfP dates are announced because the one-on-one matching depends on them.

Reply:
```suggestion
### Open until the 27th of February 2024
```

Reply:
@MariiaRomanuik According to https://github.com/EuroPython/website/pull/454 the dates of the CfP are 14 Feb - 6 Mar so a good deadline for the one-to-one is 27th Feb!